examples/basic_storage.py

```diff
@@ -48,7 +48,7 @@ async def main():
         sp = await blockchain_client.sp.get_first_in_service_storage_provider()
         primary_sp_address = sp["operator_address"]
         
-        async with StorageClient(key_manager=key_manager, sp_endpoints=sp_endpoints) as client:
+        async with StorageClient(network_configuration=network_configuration, key_manager=key_manager, sp_endpoints=sp_endpoints) as client:
```

Comment:
Excuse my python illiteracy, does argument positioning matter? Should we add the new argument to the end of the signature?

Reply:
The behavior depends on whether the arguments are Keyword-Only or Positional-Only.
In this specific case, we are using Keyword-Only arguments. With Keyword-Only arguments, it is not an issue to add a new argument at the beginning of the function signature.